_posts/2018-05-20-Y-The-Most-Beautiful-Idea-in-Computer-Science.md

```diff
@@ -0,0 +1,611 @@
+---
```

Comment:
I use https://github.com/igorshubovych/markdownlint-cli in a pre-commit in a work project. Saves me a lot of time, and works with SublimeLinter via https://github.com/jonlabelle/SublimeLinter-contrib-markdownlint

Reply:
So useful! Thanks!